src/main/resources/com/mathworks/ci/RunMatlabTestsStep/help-selectByTag.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Specify the test Tag name. Test which are tagged with specified name will be added to test suite and will be considered to run.</p>
```

Comment:
Specify the test tag used to select test suite elements. To generate a test suite, MATLAB uses only the test elements with the specified tag.

Don't we need an example here similar to the one for selectByFolder? For example:
<p><b>Example:</b> 'FeatureA'</p>

Reply:
Done.